src/bin/nydusd/main.rs

```diff
@@ -715,8 +724,17 @@ fn main() -> Result<()> {
     // Safe to unwrap because it has default value and possible values are defined
     let level = args.value_of("log-level").unwrap().parse().unwrap();
     let apisock = args.value_of("apisock");
+    let rotation_size = match args.value_of("log-rotation-size").unwrap().parse::<u64>() {
+        Ok(size) => size,
+        Err(error) => {
+            return Err(Error::new(
+                ErrorKind::InvalidData,
+                format!("Invalid log rotation size: {}", error),
+            ));
+        }
```

Comment:
match the result is fine. But a neater style will be:
```
    // Safe to unwrap since it has a default value.
    let rotation_size = args
        .value_of("log-rotation-size")
        .unwrap()
        .parse::<u64>()
        .map_err(|e| einval!(format!("Invalid log rotation size: {}", e)))?;
```

Reply:
Ok, I learned.